discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+            exists = (
+                session.query(CustomCommand).filter(CustomCommand.name == name).first()
+            )
+            if not exists:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            session.delete(exists)
+            session.commit()
+
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Command `{name}` removed",
+                    colour=Colour.green(),
+                )
+            )
+
+    @group.command(name="removedbbackup", description="Remove a database backup")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(db_filename="Name of backup file")
+    async def removedbbackup(self, interaction: Interaction, db_filename: str):
+        if not db_filename.startswith(config.DB_NAME) or not db_filename.endswith(
+            ".db"
+        ):
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Filename must be of the format {config.DB_NAME}_{{date}}.db",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+            return
+
+        try:
+            os.remove(db_filename)
+        except Exception as e:
+            _log.exception(f"Caught Exception in removedbbackup: {e}")
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"Failed to remove DB Backup {db_filename}",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+        else:
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"DB backup {db_filename} removed",
+                    colour=Colour.green(),
+                )
+            )
+
+    @group.command(name="restart", description="Restart the bot")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    async def restart(self, interaction: Interaction):
+        await interaction.response.send_message(
+            embed=Embed(
+                description="Restarting bot...",
+                colour=Colour.blue(),
+            )
+        )
+        os.execv(sys.executable, ["python", "-m", "discord_bots.main"])
+
+    @group.command(name="setbias", description="Set team bias")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(member="Member to bias", amount="Bias value")
+    async def setbias(self, interaction: Interaction, member: Member, amount: float):
```

Comment:
Since these are "meme" functions, we may just want to leave them as prefix command or (if there's a way to) hide them. Not really sure, since they just pollute the command list

Reply:
I would rather not leave anything as prefix / context commands, as the direction from discord itself is to move away from context commands entirely.
https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Privileged-Intent-FAQ

Reply:
If the prefix command @mentions the bot or is used inside of the bot's DM, then it doesn't required the message content privileged intent.